src/app-layer-detect-proto.c

```diff
@@ -1962,6 +1962,9 @@ int AppLayerProtoDetectConfProtoDetectionEnabledDefault(
         if (node == NULL) {
             SCLogDebug("Entry for %s not found.", param);
             if (default_enabled) {
+                SCLogWarning(SC_ERR_CONF_YAML_ERROR,
+                        "Proto entry not found in suricata.yaml. Enabling anyway."
+                        "Behavior to be discontinued in Suricata 7. Update configuration.");
```

Comment:
@victorjulien do you think this is appropriate? Should the message or log level or both be changed?